lib/pbench/server/api/resources/pbench_users.py

```diff
@@ -265,29 +356,40 @@ def get(self, username):
         """
         # get the auth token
         auth_header = request.headers.get("Authorization")
-        auth_token = ""
-        if auth_header:
-            try:
-                auth_token = auth_header.split(" ")[1]
-            except IndexError:
-                self.app.logger.info(f"Bearer token malformed: {auth_header}")
-                abort(401, message="Bearer token malformed")
 
-        if auth_token:
-            user_id = UserModel.decode_auth_token(auth_token, self.app)
-            user = UserModel.query.filter_by(username=username).first()
-            if user.id != user_id:
-                self.app.logger.info(
-                    f"User_id decoded from the auth token {user_id} is different for the user's id {user.id}"
-                )
-                abort(403, message="Authentication token does not belong to the user")
+        if not auth_header:
+            self.app.logger.error(f"Auth token not provided during logout")
+            abort(
+                400,
+                message="Please add 'Authorization' token as Authorization: Bearer <JWT_Auth_token>",
+            )
+
+        try:
+            auth_token = auth_header.split(" ")[1]
+        except IndexError:
+            self.app.logger.error(f"Malformed Auth header during logout")
+            abort(
+                400,
+                message="Malformed Auth header, please add 'Authorization' token as Authorization: Bearer <JWT_Auth_token>",
+            )
 
+        try:
+            user_id = UserModel.decode_auth_token(auth_token, self.app)
             if not isinstance(user_id, str):
                 user = UserModel.query.filter_by(id=user_id).first()
+                if user.username != username:
+                    self.app.logger.info(
+                        f"Username retrieved from the auth token {user_id} is different for the username provided"
```

Comment:
Hmm... not sure how many like this I overlooked, but ideally log messages shouldn't be pre-formatted. Let the logger do lazy substitution when the message is actually selected. Not a big deal for `info` which is usually enabled, but it's best to be consistent.